Makefile

```diff
@@ -30,6 +30,7 @@ CLUSTER_NAME ?= test-gslb
 CLUSTER_GEO_TAGS ?= eu us cz af ru ap uk ca
 CHART ?= k8gb/k8gb
 CLUSTER_GSLB_NETWORK = k3d-action-bridge-network
+CLUSTER_GSLB_GATEWAY = $(shell docker network inspect ${CLUSTER_GSLB_NETWORK} -f '{{ (index .IPAM.Config 0).Gateway }}')
```

Comment:
In case of nonexistent `k3d-action-bridge-network` this evaluation will fail:
```
❯ make                  
Error: No such network: k3d-action-bridge-network
```
I think it makes sense to have it as text for a shell command and postpone the evaluation until the make target.

Reply:
@somaritane fixed